endpoints.json

```diff
@@ -62,6 +62,20 @@
       ]
     }
   },
+  "POST /api/articles/:article_id/comments": {
+    "description": "adds a comment that is related to one of the articles",
+    "queries": [],
+    "exampleResponse": {
```

Comment:
Dont forget to include the exampleBody of what we expect the user to send us to actually create a new comment.